core/auth/fake/interfaces/identifier.go

```diff
@@ -0,0 +1,127 @@
+package interfaces
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"flamingo.me/flamingo/v3/core/auth"
+	"flamingo.me/flamingo/v3/core/auth/fake/domain"
+	"flamingo.me/flamingo/v3/framework/config"
+	"flamingo.me/flamingo/v3/framework/flamingo"
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+type (
+	// Identifier is the fake Identifier implementation
+	Identifier struct {
+		responder     *web.Responder
+		broker        string
+		reverseRouter web.ReverseRouter
+		eventRouter   flamingo.EventRouter
+	}
+
+	fakeConfig struct {
+		Broker           string                `json:"broker"`
+		LoginTemplate    string                `json:"loginTemplate"`
+		ValidatePassword bool                  `json:"validatePassword"`
+		ValidateOtp      bool                  `json:"validateOtp"`
+		UsernameFieldID  string                `json:"usernameFieldId"`
+		PasswordFieldID  string                `json:"passwordFieldId"`
+		OtpFieldID       string                `json:"otpFieldId"`
+		UserConfig       map[string]userConfig `json:"userConfig"`
+	}
+
+	userConfig struct {
+		Password string `json:"password"`
+		Otp      string `json:"otp"`
+	}
+)
+
+// FakeAuthURL - URL to fake login page
+const FakeAuthURL string = "/core/auth/fake/:broker"
+
+var (
+	_ auth.RequestIdentifier = (*Identifier)(nil)
+	_ auth.WebCallbacker     = (*Identifier)(nil)
+	_ auth.WebLogouter       = (*Identifier)(nil)
+
+	identifierConfig map[string]fakeConfig
```

Comment:
Why this global variable? This is not safe for use in multiple flamingo config areas